### PR TITLE
Set much larger ulimits

### PR DESCRIPTION
**Pull request by paololim**

### CODE REVIEW

File: api/app/aws/EC2ContainerService.scala [hunk 1]
```diff
@@ -335,6 +335,14 @@ case class EC2ContainerService @javax.inject.Inject() (
                   .withName(containerName)
                   .withImage(imageName + ":" + imageVersion)
                   .withMemory(settings.containerMemory)
+                  .withUlimits(
+                    Seq(
+                      new Ulimit()
+                        .withName(UlimitName.Nofile)
+                        .withSoftLimit(1024000)
```
Comment (mbryzek) [inline api/app/aws/EC2ContainerService.scala]: if these values need to be the same here and in AutoScalingGroup - might be worth defining as a val

File: api/app/aws/AutoScalingGroup.scala [hunk 2]
```diff
@@ -217,7 +217,9 @@ class AutoScalingGroup @javax.inject.Inject() (
       """curl -o /tmp/sumo.sh https://collectors.sumologic.com/rest/download/linux/64""",
       """chmod +x /tmp/sumo.sh""",
       """export PRIVATE_IP=$(curl http://169.254.169.254/latest/meta-data/local-ipv4)""",
-      s"""sh /tmp/sumo.sh -q -Vsumo.accessid="${sumoId}" -Vsumo.accesskey="${sumoKey}" -VsyncSources="/etc/sumo/sources.json" -Vcollector.name="${id}-""" + "$PRIVATE_IP\""
+      s"""sh /tmp/sumo.sh -q -Vsumo.accessid="${sumoId}" -Vsumo.accesskey="${sumoKey}" -VsyncSources="/etc/sumo/sources.json" -Vcollector.name="${id}-""" + "$PRIVATE_IP\"",
+      """echo '* soft nofile 1024000' >> /etc/security/limits.conf""",
+      """echo '* hard nofile 1024000' >> /etc/security/limits.conf"""
```
Comment (pnguyen) [inline api/app/aws/AutoScalingGroup.scala]: Add another line to "service docker restart" so it can pick up the new file limit changes.

Comment (paololim) [inline api/app/aws/AutoScalingGroup.scala]: I made a change to the task definition creation that allows you to specify the ulimit of the docker container - does that count?

Comment (pnguyen) [inline api/app/aws/AutoScalingGroup.scala]: I've read some threads where people had mixed results with passing the limits to the container, so I guess we'd just have to try and see. But I do think a restart of the docker daemon would make sure the daemon always sees the new limits all the time and not just through the task definition.

Comment (paololim) [inline api/app/aws/AutoScalingGroup.scala]: Ack - does that also restart the ecs agent?

Comment (pnguyen) [inline api/app/aws/AutoScalingGroup.scala]: No, it wouldn't restart the ecs agent, but I don't think you need to restart ecs.

**Pull request merged**